fms/fms.F90

```diff
@@ -300,6 +300,10 @@ module fms_mod
    module procedure string_from_integer
    module procedure string_from_real
 end interface
+interface fms_c2f_string
```

Comment:
Should have doxygen comments for the added public interface to show up (needs the `@ingroup fms_mod` like the string interface above)

Reply:
@rem1776 Thanks for pointing that out